.github/workflows/release.yml

```diff
@@ -250,62 +250,36 @@ jobs:
 
       - name: Catalog - update veneer file
         run: |
-          cd everest-catalog
+          cd everest-catalog/tools
 
           # configure userdata for commits
           git config --global user.email "everest-ci@percona.com"
           git config --global user.name "Everest RC CI triggered by ${{ github.actor }}"
 
-          CURRENT_STABLE_VERSION=$(yq 'select(.name == "stable-v0").entries[-1].name' veneer/everest-operator.yaml | sed 's/^everest-operator.v//')
-          CURRENT_FAST_VERSION=$(yq 'select(.name == "fast-v0").entries[-1].name' veneer/everest-operator.yaml | sed 's/^everest-operator.v//')
-
-          if [[ -z "$CURRENT_STABLE_VERSION" ]]; then
-            echo "CURRENT_STABLE_VERSION is required"
-            exit 1
-          fi
-
-          if [[ -z "$CURRENT_FAST_VERSION" ]]; then
-            echo "CURRENT_FAST_VERSION is required"
-            exit 1
-          fi
-
-          echo "CURRENT_STABLE_VERSION=$CURRENT_STABLE_VERSION"
-          echo "CURRENT_FAST_VERSION=$CURRENT_FAST_VERSION"
-
-          cp veneer/everest-operator.yaml veneer/everest-operator-original.yaml
+          cp ../veneer/everest-operator.yaml ../veneer/everest-operator-original.yaml
           
-          if [[ $env.IS_RC ]]; then
-            go run ./tools/ \
-              --veneer-file veneer/everest-operator-original.yaml \
-              --version-type ${{ env.RELEASE_TYPE }} \
-              --channel fast-v0 \
-              --new-version ${{ env.VERSION }} \
-              --current-version "$CURRENT_FAST_VERSION" \
-              \
-              >| veneer/everest-operator.yaml
+          if [[ $IS_RC == 1 ]]; then
+            go run . \
+              --veneer-file ../veneer/everest-operator-original.yaml \
+              --channel fast-v0  \
+              --new-version ${{ env.VERSION }} 
           else
-            go run ./tools/ \
-              --veneer-file veneer/everest-operator-original.yaml \
-              --version-type ${{ env.RELEASE_TYPE }} \
+            go run . \
+              --veneer-file ../veneer/everest-operator-original.yaml \
               --channel stable-v0 \
               --new-version ${{ env.VERSION }} \
-              --current-version "$CURRENT_STABLE_VERSION" \
-              \
-              >| veneer/everest-operator.yaml
-
-            go run ./tools/ \
-              --veneer-file veneer/everest-operator-original.yaml \
-              --version-type ${{ env.RELEASE_TYPE }} \
+              >| ../veneer/everest-operator.yaml
+          
+            cp ../veneer/everest-operator.yaml ../veneer/everest-operator-original.yaml
```

Comment:
I'm a bit confused, in the IS_RC conditional you don't output it to the non original file and here you do and then you copy it to the original

Reply:
Thanks for drawing attention to it. After testing I forgot to put back the writing of the command output to the original file. So we always should write to the original file

Reply:
Ok now that part looks good but do we actually need 2 files here? Couldn't we always use `../veneer/everest-operator.yaml` instead of copying it back and forth to `../veneer/everest-operator-original.yaml`?

Reply:
The temporary file was needed bc the command reads the file and writes the output of itself to the same file. The operational system doesn't handle it well and at some point the file become empty. 
Reworked the update tool so that we won't need a temporary file, ptal https://github.com/percona/everest-catalog/pull/33